### PR TITLE
faq/troubleshooting: when are device types discontinued

### PR DESCRIPTION
**Pull request by alexgg**
Thread: https://balena.zulipchat.com/#narrow/stream/345746-aspect.2Fproduct/topic/balenaFin.20discontinuation
Arch call: https://drive.google.com/file/d/1hl1D3q9AwpUrp7XxgQk7HoGDInWL_Z0J/view?usp=drive_web

### CODE REVIEW

File: pages/faq/troubleshooting/faq.md [hunk 1]
```diff
@@ -122,6 +123,10 @@ The {{ $names.company.lower }} device supervisor needs to be able to access our
 
 Generally, we try to follow good OPSEC practices for our systems. We support 2FA for user accounts and force all the connections to be over HTTPS. More details on our approach can be found on our [security page][security].
 
+##### When are device types discontinued?
+
+Device types are [discontinued](#what-does-it-mean-when-a-device-type-is-discontinued) in {{ $names.cloud.lower }} when the manufacturer discontinues support for them. This usually happens either when the device is end-of-life and manufacturing is stopped, or when software updates by the manufacturer stop happening and updating the device type's software is no longer possible. Discontinued device types will still receive support for up to two years after the release date of the last {{ $names.os.lower }} version that supports that device type.
```
[the remark this answers is not in the document]
Comment (rcooke-warwick) [reply pages/faq/troubleshooting/faq.md:128]: also @alexgg @vipulgupta2048 the dashboard warning takes you to the docs that describes a discontinued device type: https://docs.balena.io/faq/troubleshooting/faq/#what-does-it-mean-when-a-device-type-is-discontinued

[the remark this answers is not in the document]
Comment (rcooke-warwick) [reply pages/faq/troubleshooting/faq.md:128]: Also, couldn't we just amend this part to the "What does it mean when a device type is discontinued?" heading below?

File: pages/faq/troubleshooting/faq.md [hunk 2]
```diff
@@ -5,29 +5,30 @@ title: FAQs
...
+ - [If the device is installed behind a proxy/firewall and can’t be reachable on internet via direct connection, what are the pitfalls?](#if-the-device-is-installed-behind-a-proxyfirewall-and-cant-be-reachable-on-internet-via-direct-connection-what-are-the-pitfalls)
+ - [How do you secure your own cloud to prevent malicious attack which may allow attacker to break-in to our systems?](#how-do-you-secure-your-own-cloud-to-prevent-malicious-attack-which-may-allow-attacker-to-break-in-to-our-systems)
+ - [When are device types discontinued?](#when-are-device-types-discontinued)
+ - [What does it mean when a device type is discontinued?](#what-does-it-mean-when-a-device-type-is-discontinued)
+ - [I have a device that is not on the supported devices list. Can it run on {{ $names.company.lower }}?](#i-have-a-device-that-is-not-on-the-supported-devices-list-can-it-run-on--namescompanylower-)
+ - [What to keep in mind when choosing power supply units?](#what-to-keep-in-mind-when-choosing-power-supply-units)
+ - [Does {{ $names.company.lower }} have access to my device, source code and images?](#does--namescompanylower--have-access-to-my-device-source-code-and-images)
```
Comment (vipulgupta2048) [inline pages/faq/troubleshooting/faq.md:32]: Your editor auto indented this, we need the index to show this as table of contents. Please do revert this

File: pages/faq/troubleshooting/faq.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (vipulgupta2048) [reply pages/faq/troubleshooting/faq.md:128]: Both suggestions are very good ^
I would consider adding a paragraph on `what this means for you?`

File: pages/faq/troubleshooting/faq.md (hunk 2 above)
Comment (vipulgupta2048) [reply pages/faq/troubleshooting/faq.md:32]: You are indeed correct, thanks for fixing this. I don't know when exactly this broke. 
Preview here: https://ce23a6e8.balenacloud-docs.pages.dev/faq/troubleshooting/faq/

File: pages/faq/troubleshooting/faq.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexgg) [reply pages/faq/troubleshooting/faq.md:128]: thanks @rcooke-warwick I will merge your suggestions with the output of the incoming arch call about this topic

[the remark this answers is not in the document]
Comment (floion) [reply pages/faq/troubleshooting/faq.md:128]: I think we also need to be a little more specific on what we mean by users being entitled to balena support for up to 2 years? Does this mean we work on solving OS / BSP issues on the discontinued hw or issues that are surfaced on that discontinued hw but are not related to BSP but rather expose bugs in the platform?

[the remark this answers is not in the document]
Comment (alexgg) [reply pages/faq/troubleshooting/faq.md]: I have updated the text after the arch call, please review.

**Comment by alexgg** [comment]
lgtm

**Pull request merged**